lib/WP_Auth0_Settings_Section.php

```diff
@@ -65,8 +65,5 @@ public function init_menu() {
 		}
 	}
 
-	// TODO: deprecate
-	public function redirect_to_help() {
-
-	}
+	public function redirect_to_help() {}
```

Comment:
is it worthy to add the `deprecated` docblock here as well?

Reply:
I'm not entirely sure what to do with it yet. Serves no purpose but is used in a hook above. Will address when this class gets a once-over 👍